sdk/mediaservices/Azure.Media.Analytics.Edge/src/Customization/MediaGraphTopologySetRequestBody.cs

```diff
@@ -0,0 +1,8 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Media.Analytics.Edge.Models
+{
+    internal partial class MediaGraphTopologySetRequestBody
```

Comment:
Why do you need this customization?

Reply:
We want the class to be internal which is why we have this customization